telluric/vectors.py

```diff
@@ -301,24 +301,30 @@ def to_geojson(self, filename):
         with open(filename, 'w') as fd:
             json.dump(self.to_record(WGS84_CRS), fd)
 
+    @classmethod
+    def empty(cls):
+        return cls(GeometryCollection(), DEFAULT_CRS)
+
     @classmethod
     def point(cls, x, y, crs=DEFAULT_CRS):
-            return cls(Point((x, y)), crs)
+        return cls(Point((x, y)), crs)
 
     @classmethod
     def line(cls, points, crs=DEFAULT_CRS):
-            return cls(LineString(points), crs)
+        return cls(LineString(points), crs)
 
     @classmethod
-    def from_bounds(cls, *, xmin, ymin, xmax, ymax, crs=DEFAULT_CRS):
-        """Creates GeoVector object from bounds.
+    def polygon(cls, shell, holes=None, crs=DEFAULT_CRS):
```

Comment:
should this be `from_polygon` ? for me the name implies the return value is a Polygon (shapely)

Reply:
I tried to be consistent with the other two (`line` and `point`). The good thing is that, with this, the user does not even need to import shapely anymore.

Reply:
ok